drivers/storage/portworx/deployment_test.go

```diff
@@ -771,7 +746,7 @@ func TestPodSpecWithCloudStorageSpec(t *testing.T) {
 		"-max_storage_nodes_per_zone", "3",
 	}
 
-	actual, err = driver.GetStoragePodSpec(cluster)
+	actual, err = driver.GetStoragePodSpec(cluster, "nodeName1")
```

Comment:
Can we separate this out in multiple tests - 
- without any user provided `maxStorageNodesPerZone`, use the same node name, and the instances per zone increase, the `-max_storage_nodes_per_zone` should be still 2 because we have a storage node now from the previous run.
- without any user provided `maxStorageNodesPerZone`, use a new node name, and the increased instances per zone.. should have the same output
- user provided `maxStorageNodesPerZone`.. now the `-max_storage_nodes_per_zone` should change to what the user has passed.